lib/README.md

```diff
@@ -0,0 +1,75 @@
+# Health Cards
+
+![Build](https://github.com/dvci/health_cards/actions/workflows/ruby.yml/badge.svg)
+![Maintainability](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/maintainability)
+![Test Coverage](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/test_coverage)
+
+Health Cards is a Ruby gem that implements [SMART Health Cards](https://smarthealth.cards), a secure and decentralized framework that allows people to prove their vaccination status or medical test results. It is built on top of [FHIR 4](https://hl7.org/FHIR/) healthcare interoperability standards and converts medical data into a [JWS](https://en.wikipedia.org/wiki/JSON_Web_Signature) that is then embedded into a QR code.
+
+This library conforms to the [SMART Health Cards: Vaccination & Testing Implementation Guide](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/), and a reference implementation in rails can be found [here](https://github.com/dvci/health_cards).
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'health_cards'
+```
+
+And then execute:
+
+```
+ $ bundle install
+```
+
+Or install it yourself as:
+
+```
+ $ gem install health_cards
+```
+
+## Usage
+
+```ruby
+# Create issuer
+private_key = HealthCards::Key.generate_key
+issuer = HealthCards::Issuer.new(key: private_key)
+
+# Create health card
+medical_data = FHIR::Bundle.new # populate with patient data
```

Comment:
Same as above, I think "clinical data" is used more than "medical data", even though they're almost synonymous

Reply:
fixed

Reply:
I'm seeing the change to "clinical" in the overview description, but not here in the code example.

Reply:
You'll want to also change the parameter on line 43 to `clinical_data`. Then I would also make these changes in the root README so that the two are consistent.

Reply:
Ended up removing the lib/README